.github/workflows/sanitize-runbook.yml

```diff
@@ -23,15 +27,14 @@ jobs:
           /usr/bin/env python -m pip install PyGithub
       
       - name: PR File List
-        uses: jitterbit/get-changed-files@v1
+        uses: tj-actions/changed-files@v35
         id: files
 
-  
       - name: Run Sanitize
         run: |
-          echo "Running sanitize script on ${{ steps.files.outputs.all }}"
-          for changed_file in ${{ steps.files.outputs.all }}; do
-            /usr/bin/env python ./sanitize.py ${changed_file}
+          echo "Running sanitize script on ${{ steps.files.outputs.all_changed_files }}"
+          for changed_file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            /usr/bin/env python ./sanitize.py $changed_file
           done
       
       - name: Committing back changes
```

Comment:
@abhishek-unskript so are you removing this step from the action?

Reply:
I can take care of it in the next commit...